src/ring_theory/fractional_ideal.lean

```diff
@@ -101,71 +99,92 @@ open submodule
 variables {R : Type*} [comm_ring R] {S : submonoid R} {P : Type*} [comm_ring P]
 variables [algebra R P] [loc : is_localization S P]
 
+/-- Map a fractional ideal `I` to a submodule by forgetting that `∃ a, a I ⊆ R`.
+
+This coercion is typically called `coe_to_submodule` in lemma names
+(or `coe` when the coercion is clear from the context),
+not to be confused with `is_localization.coe_submodule : ideal R → submodule R P`
+(which we use to define `coe : ideal R → fractional_ideal S P`, and
+is referred to `coe_ideal` in theorem names).
```

Comment:
```suggestion
not to be confused with `is_localization.coe_submodule : ideal R → submodule R P`
(which we use to define `coe : ideal R → fractional_ideal S P` which
is referred to `coe_ideal` in theorem names).
```
otherwise it makes it sound like `coe_submodule` is referred to as `coe_ideal`.